firefox-ios/Ecosia/Account/Auth/Auth0SessionTokenRequest.swift

```diff
@@ -0,0 +1,49 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+/// Struct for encoding the JSON request body
+struct TokenRequest: Codable {
+    let grantType: String
+    let subjectToken: String
+    let subjectTokenType: String
+    let requestedTokenType: String
+    let clientId: String
+}
+
+/// Struct for decoding the JSON response
+struct TokenResponse: Codable {
+    let accessToken: String
+}
```

Comment:
Similarly here, maybe it makes sense to move it to a specific separate file? It's not used inside here anyway... It would then follow a similar pattern to `SingularConversionValueResponse` (even if this is much simpler).